.gitlab/ci/on-push.yml

```diff
@@ -7,6 +7,173 @@
   - echo "======= Download configuration ========"
   - ./Tests/scripts/download_demisto_conf.sh  >> $ARTIFACTS_FOLDER/logs/installations.log
 
+.notify-slack:
+  after_script:
+    - echo "=====Notify Slack====="
+    - |
+      if [ -n "${SLACK_ONLY_IF}" ]; then
+        ./Tests/scripts/slack_notifier.sh "$SLACK_TEST_TYPE" "$SLACK_ENV_RESULTS" "$SLACK_CHANNEL"
+      else
+        echo "Skipping Slack"
+      fi
+
+run-unittests-and-lint:
+  needs: []
+  stage: unittests-and-validations
+  rules:
+    - if: '$FORCE_BUCKET_UPLOAD != "true"'
+  artifacts:
+    expire_in: 48 hrs
+    paths:
+      - /builds/xsoar/content/unit-tests
+      - /builds/xsoar/content/artifacts/*
+  extends:
+    - .default-job-settings
+    - .notify-slack
+  variables:
+    SLACK_TEST_TYPE: "unittests"
+    SLACK_ENV_RESULTS: $ENV_RESULTS_PATH
+    SLACK_ONLY_IF: $BUCKET_UPLOAD
```

Comment:
When you overwrite the `variable` key, do you still have the global variable too? or does if overwrites them?

Reply:
it overwrites them